tasks/release.js

```diff
@@ -0,0 +1,64 @@
+try {
+    require( 'shelljs/global' );
+    const colors = require( 'colors' );
+    const confirm = require( 'confirm-simple' );
+    const archiver = require( 'archiver' );
+    const fs = require( 'fs' );
+} catch( e ) {
+    console.error( 'Please run `npm install` first.'.red );
```

Comment:
Seems like it's not a good idea to use `.red` when the failure could be due to not npm installing the colors library.